Dockerfile

```diff
@@ -1,13 +1,13 @@
 # Use an official Python base image from the Docker Hub
 FROM python:3.11-slim
 
-# Install git
-RUN apt-get -y update
-RUN apt-get -y install git chromium-driver
-
-# Install Xvfb and other dependencies for headless browser testing
-RUN apt-get update \
-    && apt-get install -y wget gnupg2 libgtk-3-0 libdbus-glib-1-2 dbus-x11 xvfb ca-certificates
+# Install packages
+RUN apt-get update && apt-get install -y \
```

Comment:
I agree this should be cleaned up but you could split the `apt-get update` from the `apt-get install` in order to make optimal use of the docker cache when building

Reply:
The only thing that will trigger a rebuild of this layer is pulling a new version of `python:3.10-slim`. Considering that, it doesn't matter whether `update` and `install` are combined or executed in separate steps, right?